app.rb

```diff
@@ -0,0 +1,60 @@
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @peoples = []
+  end
+
+  def list_all_books
+    @books.each { |book| puts "Title:- #{book.title}   Author:- #{book.author} \n" }
+  end
+
+  def list_all_peoples
+    @peoples.each do |people|
+      puts "[#{people.class.name}] Name:- #{people.name} ID:- #{people.id} Age:- #{people.age} \n"
+    end
+  end
+
+  def create_teacher(age, specialization, name, parent_permission)
+    @peoples << Teacher.new(age, specialization, name: name, parent_permission: parent_permission)
+  end
+
+  def create_student(age, name, parent_permission)
+    @peoples << Student.new(age, name: name, parent_permission: parent_permission)
+  end
+
+  def create_book
+    puts 'Title:-'
+    title = gets.chomp
+    puts 'Author:-'
+    author = gets.chomp
+    @books << Book.new(title, author)
+  end
+
+  def create_rental
+    puts 'Select a book from the following list by number'
+    @books.each.with_index { |book, idx| puts "#{idx}) Book #{book.title} by #{book.author}" }
+    book_index = gets.chomp.to_i
+    puts 'Select a person from the following list by number(not ID)'
+    @peoples.each.with_index do |person, idx|
+      puts "#{idx}) [#{person.class.name}] Name #{person.name}, ID #{person.id}, Age #{person.age}"
+    end
+    person_index = gets.chomp.to_i
+    puts 'Date:-'
```

Comment:
- Hey, freat job implementing the changes requested by the previous reviewer, however, in the console, kindly indicate the date format as this leaves users confused about how to input the date. it could be an example like `yyyy-mm-dd or dd-mm-yyyy`. please, let's update this also to make this part of your code readable